config.yaml

```diff
@@ -13,6 +13,14 @@ keywords:    express
 # list of projects, this will be sorted by name;
 # every package is either string or object
 projects:
+ - cors
+ - body-parser
```

Comment:
This one and several others are already in the list.

Reply:
Oh oops, I was eyeballing it, must have missed it the first time around.  Will remove in the morning.